src/misc/ListModel.ts

```diff
@@ -328,8 +328,17 @@ export class ListModel<ElementType extends ListElement> {
 	}
 
 	/** An element was added to the selection. If multiselect was not on, add previous single selection and newly added selected item to the selection. */
-	onSingleInclusiveSelection(item: ElementType): void {
+	onSingleInclusiveSelection(item: ElementType, clearSelectionOnMultiSelectStart?: boolean): void {
+		// If isn't in MultiSelect, we discard all previous items
```

Comment:
Everything looks good, so sorry to nit-pick on the comment. "If isn't in MultiSelect" isn't quite grammatically correct, I would phrase it as "If it isn't in MultiSelect" or "If not in MultiSelect"

Reply:
Thanks for the insight